Snowflake.Data/Core/Session/SFSessionProperty.cs

```diff
@@ -254,7 +254,7 @@ internal static SFSessionProperties ParseConnectionString(string connectionStrin
             }
 
             ValidateAuthenticator(properties);
-            DisableConnectionPoolingForExternalBrowser(properties);

```

Comment:
Let's add parameter to this ParseConnection method (ln 180) so this Disable method won't be called for not SessionPool.IsMultiplePoolsVersion

Reply:
Irrelevant, this is happening only in new pool